src/metafiles/cvs.rs

```diff
@@ -1,19 +1,30 @@
 use std::{borrow::Cow, fmt, str::FromStr};
 
 use miette::Diagnostic;
-use once_cell::sync::Lazy;
 use semver::Version;
 use serde::{Deserialize, Deserializer, Serialize, Serializer};
 use thiserror::Error;
 use url::Url;
 
+use crate::cratesio_url;
+
 #[derive(Clone, Debug, Ord, PartialOrd, Eq, PartialEq)]
 pub struct CrateVersionSource {
     pub name: String,
     pub version: Version,
     pub source: Source,
 }
 
+impl From<&super::binstall_v1::MetaData> for CrateVersionSource {
+    fn from(metadata: &super::binstall_v1::MetaData) -> Self {
+        super::CrateVersionSource {
```

Comment:
I think that `super::` is extraneous?

Reply:
Nope, we need that here for the code to compile.